benchmarking/data/dataYao_target=1.json

```diff
@@ -0,0 +1 @@
+{"Y":{"nqubits":[4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25],"times":[131.89941520467838,173.81523272214386,240.48899755501222,338.2877358490566,561.3858695652174,1023.4,1916.0,3743.5,7317.0,14542.0,29106.0,57816.0,115208.0,230277.0,460258.0,919214.0,2.045396e6,4.364127e6,8.798101e6,1.7614862e7,3.5336555e7,7.0376468e7]},"X":{"nqubits":[4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25],"times":[110.81473456121343,120.08690869086908,150.84859154929578,191.19814241486068,274.4323432343234,429.61306532663315,753.6779661016949,1447.2,2734.1111111111113,5321.166666666667,10994.0,27995.0,55568.0,111473.0,223481.0,445963.0,1.39466e6,3.393371e6,6.783388e6,1.3762345e7,2.764668e7,5.4447295e7]},"T":{"nqubits":[4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25],"times":[139.8916083916084,161.53907284768212,211.86666666666667,304.584,482.8820512820513,846.0588235294117,1570.4,3061.375,5953.666666666667,11797.0,23479.0,46881.0,93422.0,186793.0,373317.0,746479.0,1.781669e6,4.00497e6,7.99523e6,1.6112408e7,3.2225915e7,6.4169955e7]},"CNOT":{"nqubits":[4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25],"times":[162.2984693877551,177.74894217207336,210.96666666666667,271.65714285714284,394.67164179104475,622.3430232558139,1086.4,2068.1111111111113,3931.75,7679.25,15318.0,30683.0,60664.0,120964.0,241189.0,486569.0,1.440134e6,3.545929e6,7.1459e6,1.446229e7,2.8683109e7,5.6771631e7]},"H":{"nqubits":[4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25],"times":[165.625814863103,209.8183486238532,303.712,495.37628865979383,878.5961538461538,1634.9,3144.125,6196.6,12277.0,24363.0,48527.0,96771.0,193116.0,386009.0,771554.0,1.551886e6,3.196181e6,6.545071e6,1.313185e7,2.6207816e7,5.2534204e7,1.06917705e8]}}
```

Comment:
As this file is specific to the developer machine, I would remove it until we have a standardized machine to compare the benchmarks on.